src/renderer/state/upload/logics.ts

```diff
@@ -225,6 +225,10 @@ const initiateUploadLogic = createLogic({
       try {
         await fms.upload(upload);
       } catch (error) {
+        if (error?.response?.status === 503) {
+          window.alert("The upload service is undergoing scheduled maintenance (503). Please try again later.");
```

Comment:
I would leave the 503 out of the message - this is aimed a user who has no use for that detail.